pkg/router/controller/status_test.go

```diff
@@ -428,3 +428,41 @@ func TestFindOrCreateIngress(t *testing.T) {
 	}
 	t.Logf("routes: %#v", route.Status.Ingress)
 }
+
+func TestLastTransitionTimeIsRFC3339(t *testing.T) {
+	p := &fakePlugin{}
+	c := testclient.NewSimpleFake(&routeapi.Route{})
+	admitter := NewStatusAdmitter(p, c, "test")
+
+	route := routeapi.Route{
+		ObjectMeta: kapi.ObjectMeta{Name: "route1", Namespace: "default", UID: types.UID("uid1")},
+		Spec:       routeapi.RouteSpec{Host: "route1.test.local"},
+	}
+
+	// Add the route and validate that the status is set.
+	err := admitter.HandleRoute(watch.Added, &route)
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+	if len(c.Actions()) != 1 {
+		t.Fatalf("unexpected actions: %#v", c.Actions())
+	}
+	action := c.Actions()[0]
+	if action.GetVerb() != "update" || action.GetResource() != "routes" || action.GetSubresource() != "status" {
+		t.Fatalf("unexpected action: %#v", action)
+	}
+	obj1 := c.Actions()[0].(ktestclient.UpdateAction).GetObject().(*routeapi.Route)
```

Comment:
You have defined `action` in L450, you can use that instead of accessing via the method again.